ldap/servers/plugins/pwdstorage/crypt_pwd.c

```diff
@@ -119,21 +109,21 @@ crypt_pw_enc_by_hash(const char *pwd, int hash_algo)
 char *
 crypt_pw_enc(const char *pwd)
 {
-    return crypt_pw_enc_by_hash(pwd, CRYPT_UNIX);
+    return crypt_pw_enc_by_hash(pwd, CRYPT_UNIX_SALT_LENGTH, NULL);
```

Comment:
Globally the change looks good.
A minor remark: 
Instead of using NULL as algo_id and checking for it in crypt_pw_enc_by_hash, you could use the empty string "" so that the slapi_ch_smprintf handles directly all the cases

Reply:
Oh yeah, good point. I can make that change from NULL to "", but my git skills are terrible. Can I still squash the new commit into the original for the PR?

Reply:
> Oh yeah, good point. I can make that change from NULL to "", but my git skills are terrible. Can I still squash the new commit into the original for the PR?

You have a few options, you can squash the commits (see https://www.port389.org/docs/389ds/contributing.html#getting-the-patch-ready)   Or you could make your change, and do "git commit -a --amend" and then do a "git push --force".  This will leave it as a single commit

Reply:
Ok, that change has been made. Thanks for the help on the git commands!